sample/event-read-fifo.c

```diff
@@ -139,6 +139,10 @@ main(int argc, char **argv)
 #else
 	/* catch SIGINT so that event.fifo can be cleaned up */
 	signal_int = evsignal_new(base, SIGINT, signal_cb, base);
+	if (signal_int == NULL) {
+		perror("evsignal_new");
```

Comment:
Should have normal error path here (via goto, see other PRs comments)

Reply:
OK, I'll fix it